lib/perl/Genome/Model/Build/DeNovoAssembly/Allpaths.pm

```diff
@@ -139,7 +139,7 @@ sub before_assemble {
         my $lib_id = join("-", @lib_ids);
         my $genomic_start = 0;
         my $genomic_end = 0;
-        my $lib_name = join("-", map{Genome::Library->get($_)->name} @lib_ids);
+        my $lib_name = join("-", map{ my $name = Genome::Library->get($_)->name; Genome::Utility::Text::sanitize_string_for_filesystem($name); } @lib_ids);
```

Comment:
Is this being used in a filename?  It looks like it's just data going into the CSV.  Is the real problem that this is writing out a CSV by hand?  Would our CSV/TSV writer correctly quote the libary names?